app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -214,7 +255,7 @@ class LuciBuildService {
   Future<bool> rescheduleBuild({
     @required String commitSha,
     @required String builderName,
-    @required push_message.Build build,
+    @required push_message.BuildPushMessage buildPushMessage,
     @required int retries,
   }) async {
     if (retries >= config.luciTryInfraFailureRetries) {
```

Comment:
shall this `retries` be removed and use `userData['retries']` instead in the condition?

Reply:
This will be removed on https://github.com/flutter/flutter/issues/60942 to avoid breaking functionality.